docs/installation.rst

```diff
@@ -316,15 +316,20 @@ Or for Python 3::
 
 .. Note:: ``redhat-rpm-config`` is required on Fedora 23, but not earlier versions.
 
-Prerequisites are installed on **Ubuntu 14.04 LTS** with::
+Prerequisites can be installed on **Ubuntu 14.04 LTS** with::
 
     $ sudo apt-get install libtiff5-dev libjpeg8-dev zlib1g-dev \
         libfreetype6-dev liblcms2-dev libwebp-dev libharfbuzz-dev libfribidi-dev \
         tcl8.6-dev tk8.6-dev python-tk
 
 Then see ``depends/install_raqm.sh`` to install libraqm.
 
-Prerequisites are installed on **Fedora 23** with::
+Prerequisites can be installed on **RedHat/CentOS 7.4** with::
+
+    $ sudo yum install libjpeg-devel libwebp-devel freetype-devel openjpeg2-devel \
+        libtiff-devel libimagequant lcms2-devel
```

Comment:
This may be why your libimagequant isn't working, since this isn't the dev package.

Reply:
@wiredfool That's a typo, I definitely have the ``devel`` package installed.